src/diffusers/loaders/lora_pipeline.py

```diff
@@ -1577,7 +1577,18 @@ def lora_state_dict(
             allow_pickle=allow_pickle,
         )
 
-        return state_dict
+        # For state dicts like
+        # https://huggingface.co/TheLastBen/Jon_Snow_Flux_LoRA
+        keys = list(state_dict.keys())
+        network_alphas = {}
+        for k in keys:
+            if "alpha" in k:
```

Comment:
Usually, the `alpha` check is just fine (saying this from previous experiences). Otherwise, it's just a malformatted LoRA.

Reply:
Yes, I agree it would probably mean that the state dict is incorrect, still wondering if a check on the alpha type could be helpful to error out early.

Reply:
Yes, that was what I was thinking about. But the one line should be `network_alphas[k] = state_dict.pop(k)`, right?

Reply:
https://github.com/huggingface/diffusers/pull/9236/commits/27405bce9ec835a6cd727c3fc24689ddb906a993 fixed here.